src/content/1.7/basics/keeping-up-to-date/upgrade.md

```diff
@@ -213,7 +213,7 @@ Modules page, under the tab “Notifications”:
 
 ## 1-click upgrade module
 
-If the manual process seems too tedious, other solutions exist to complete an upgrade.
+If the manual process seems too tedious, other solutions exists to complete an upgrade.
```

Comment:
The plural doesn't take an "s" AFAIK 🤔

Reply:
Agreed with @atomiix 👍  although I understand that the context is confusing. The only "other solution" mentioned is the 1-click upgrade module, even though the doc says there is other solutionS it only highlights one 😅